cmake/defaults/gccclangshareddefaults.cmake

```diff
@@ -42,8 +42,14 @@ endif()
 _disable_warning("deprecated")
 _disable_warning("deprecated-declarations")
 
-# Suppress unused typedef warnings emanating from boost.
-_disable_warning("unused-local-typedefs")
+# Suppress unused typedef warnings eminating from boost.
```

Comment:
Ah, and there's a typo that was introduced here -- it should be "emanating" not "eminating".